### DISCUSSION TITLE
I'm a complete noob

### DISCUSSION THREAD

**Discussion by Jax2588** [flyinghead/flycast#719] [Q&A]
This is my first attempt at installing and using an emulator. I am working with an HP laptop with a CORE i7 10th Gen and Windows 10. I realize that whatever is holding me up is probably trivial and not worth your time, but I'd appreciate any help. I don't know much about computers, and I'm dying to play Legacy Of Kain Soul Reaver.

Sincerely,
I Wish I Were Smarter

**Comment by RainaAudron** [comment]
Hey, what exactly is the issue? No need to open a ticket if there is nothing wrong with the emulator itself. You can always email me and I will help with setting it up! Contact is on [my website.](https://www.sr1hdremaster.com/contact)
